<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QLabel" name="name_lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="id_lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitbutton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(183, 255, 156)</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="age_lbl">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="location">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="location_lbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="college">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="college_lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>College</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Are you vaccinated?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="yes_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="no_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>If Yes, how many vaccine shots have you taken?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dosage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>73</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 up</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of Vaccine taken?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="vaccine_type">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Astrazeneca</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Moderna</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Johnson and Johnson's</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pfizer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Covaxin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sinopharm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coronavac</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Covishield</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sputnik</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="logo.png"/>
 </resources>
 <connections/>
</ui>
